linera-execution/tests/wasm.rs

```diff
@@ -24,10 +24,10 @@ use test_case::test_case;
 /// called correctly and consume the expected amount of fuel.
 ///
 /// To update the bytecode files, run `linera-execution/update_wasm_fixtures.sh`.
-#[cfg_attr(with_wasmer, test_case(WasmRuntime::Wasmer, 39_669, ExecutionRuntimeConfig::Synchronous; "wasmer"))]
-#[cfg_attr(with_wasmer, test_case(WasmRuntime::WasmerWithSanitizer, 40_021, ExecutionRuntimeConfig::Synchronous; "wasmer_with_sanitizer"))]
-#[cfg_attr(with_wasmtime, test_case(WasmRuntime::Wasmtime, 40_021, ExecutionRuntimeConfig::Synchronous; "wasmtime"))]
-#[cfg_attr(with_wasmtime, test_case(WasmRuntime::WasmtimeWithSanitizer, 40_021, ExecutionRuntimeConfig::Synchronous; "wasmtime_with_sanitizer"))]
+#[cfg_attr(with_wasmer, test_case(WasmRuntime::Wasmer, 55_549, ExecutionRuntimeConfig::Synchronous; "wasmer"))]
```

Comment:
If panicking is cheaper, shouldn't fuel usage have gone down here? 🤔

Reply:
In theory yes, but it's hard to measure it with this test, because it doesn't test any errors :confused:  There's a lot of variability in the fuel consumption values in this test because it's not updated on every PR, so it sometimes shows larger changes than expected when it _is_ updated :disappointed: (Which reminds to update this test on my latest PR :sweat_smile:)